libraries/standard/mqtt/tools/ci_doxygen.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
```

Comment:
Would the generic `#!/usr/bin/env python3` be better for compatibility, or does it not matter since these scripts are only intended to be used on specific systems?
```suggestion
#!/usr/bin/env python3
```

Reply:
I did not know this is better for compatibility, thank you. These scripts are not intended for a specific system.